From aeac2cb5b5e576d1713c6c83c305718920c28b63 Mon Sep 17 00:00:00 2001
From: chenzhen <chenzhen@rock-chips.com>
Date: Tue, 4 Jul 2017 11:05:21 +0800
Subject: [PATCH] ARM: dts: rk322x: change "power-mode" to "power_model" in gpu
 node

The reference code from ARM uses "power_model".
It's convenient to use the same name
for rebasing Mali DDK update from ARM and avoiding confusion.

Change-Id: I1bb87bf3d80a7dcdeedf3320751d03c4e9339ad5
Signed-off-by: chenzhen <chenzhen@rock-chips.com>
---
 arch/arm/boot/dts/rk322x.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk322x.dtsi b/arch/arm/boot/dts/rk322x.dtsi
index a358d78dda09..128b9186ef59 100644
--- a/arch/arm/boot/dts/rk322x.dtsi
+++ b/arch/arm/boot/dts/rk322x.dtsi
@@ -586,7 +586,7 @@
 		operating-points-v2 = <&gpu_opp_table>;
 		status = "disabled";
 
-		gpu_power_model: power-model {
+		gpu_power_model: power_model {
 			compatible = "arm,mali-simple-power-model";
 			voltage = <900>;
 			frequency = <500>;
-- 
2.35.3

